### ISSUE TITLE
Cannot read property 'hex' of undefined

### ISSUE THREAD

**Issue by trickyc0d3r** [tokkozhin/react-native-neomorph-shadows#40]
Hi!
Using your library on RN 0.62.
Whe I'm try to use any module of library - got error from topic

**Comment by trickyc0d3r** [comment]
Full log is `Requiring module "node_modules/react-native-neomorph-shadows/index.js", which threw an exception: TypeError: Cannot read property 'hex' of undefined`

**Comment by trickyc0d3r** [comment]
Platform : ios 14

**Comment by trickyc0d3r** [comment]
Update - that affects me only when "inner" is true in Neomorph

**Comment by cabljac** [comment]
Same issue here, using Shadow or ShadowFlex

**Comment by leandrogenas** [comment]
Yeah, same here. I'm noticing that it's happening from only importing any of the modules from this package

**Comment by leandrogenas** [comment]
It worked after I enabled debug mode, huh

**Comment by zoobibackups** [comment]
any update for this

**Comment by zoobibackups** [comment]
> Yeah, same here. I'm noticing that it's happening from only importing any of the modules from this package

any solution for this.
